apps/nowcasting-app/components/helpers/data.test.ts

```diff
@@ -0,0 +1,473 @@
+import { describe, expect, test } from "@jest/globals";
```

Comment:
Its it worth adding a bit of free text at the top here, list out the different tests that are happening, and linking them with numbers, numbers and letters below
`
1. Test for filterHistoricDataCompact
1a. filterHistoricDataCompact: "check func filters out data after prev30MinNow"
1b .


2
`
This might. help make things really clear what is tetsted

Reply:
I think this is a good idea. It'll make navigating through the tests a bit easier in addition to the description each test has.

Reply:
Have added comment blocks to delineate tests for different functions, and amended a couple of test names to make more self-explanatory, which is how I personally think tests should be in JS land – should always be able to fully collapse the tests in a file and scan down the `describe` test names to see what's what